bundles/http/Citrix/2ec16a1b-9423-45a6-8fd8-b0c42c916c0f/cb5694f4-3740-45d6-9817-44d451edb2d9.js

```diff
@@ -0,0 +1,1136 @@
+//Loaded moment-timezone library from sdk provided libraries 
+const moment = library.load('moment-timezone');
+
+//'user' object used for pagination
+let user = {
+    next_page_token:"",
+    page_size:100
+}
+
+//'meeting' object used for pagination
+let meeting = {
+    next_page_token:"",
+    page_size:200
+}
+
+//'recording' object used for pagination
+let recording = {
+    next_page_token:"",
+    page_size:50
+}
+
+// Defined the array to store the userids, that userid will be used in meetings endpoint to fetch the user specific meetings.
+let userids = [];
+
+//Defined meetingarray to store meeting id, while will be used in meeting details api call
+let meetingArray = [];
+
+// Defined the Map for storing the meetingid as key and user email as value, which is used in meeting invitations api call 
+let meetingsMap = new Map();
+
+
+
+// loading the users data
+function syncUsers(dataStore,client,context){
+    let userContext = [];
+    do{
+        let request = client.fetchSync(`/users?next_page_token=${user.next_page_token}&page_size=${user.page_size}`);
+        // console.log(`users status : ${request.status}`);
+        if(!request.ok){
+            throw new Error(request.jsonSync());
+        }
+        let response = request.jsonSync();
+        Object.entries(response).forEach(([key,value]) =>{
+            if(key === 'next_page_token'){
+                user.next_page_token=value // 
+            }
+            else if(key === 'users'){
+                let usersData = value.map(userData =>{
+                    userContext.push(userData.id);
+                    let userObj = {
+                        "userid":userData.id,
+                        "first_name":userData.first_name,
+                        "last_name":userData.last_name,
+                        "email":userData.email,
+                        "timezone":userData.timezone,
+                        "last_login_time":moment.utc(userData.last_login_time).toDate()

```

Comment:
Nit: datetime from Zoom API comes in UTC already, so it is not necessary to use moment.utc() - this function can be used when you do not have timezone.
IMHO here could be just
```suggestion
                        "last_login_time": new Date(userData.last_login_time)
```